scripts/core/platform.yml

```diff
@@ -168,6 +168,28 @@ params:
             [out] pointer to the handle of the platform object created.
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Get the adapter specific compiler backend option from a generic frontend option."
+class: $xPlatform
+name: GetBackendOption
+decl: static
+details:
+    - "The string returned via the ppAdapterOption is a NULL terminated C style string."
+    - "The string returned via the ppAdapterOption is thread local."
+    - "The memory in the string returned via the ppMessage is owned by the adapter."
+    - "The application may call this function from simultaneous threads."
+    - "The implementation of this function should be lock-free."
+params:
+    - type: $x_platform_handle_t
+      name: hPlatform
+      desc: "[in] handle of the platform instance."
+    - type: const char*
+      name: pFrontendOptions
+      desc: "[in] string containing the frontend option."
+    - type: const char**
+      name: ppAdapterOption
+      desc: "[out] returns the correct adapter specific option based on the frontend option."
```

Comment:
Are adapter/platform being used interchangeably?

Reply:
Adapter/plugin are interchangeable. Platform is generic to the unified-runtime I believe.

Reply:
👍 understood